src/Native/gen-buildsys-win.bat

```diff
@@ -27,14 +26,15 @@ for /f "delims=" %%a in ('powershell -NoProfile -ExecutionPolicy ByPass "& .\pro
 popd
 
 :DoGen
-"%CMakePath%" "-DCMAKE_BUILD_TYPE=%CMAKE_BUILD_TYPE%" "-DCMAKE_INSTALL_PREFIX=%__CMakeBinDir%" "-DMKL_LIB_PATH=%MKL_LIB_PATH%" -G "Visual Studio %__VSString%" -B. -H%1
+if /i "%3" == "x64"     (set __ExtraCmakeParams=%__ExtraCmakeParams% -A x64)
```

Comment:
We should also support x86. The VS 2019 project generator doesn't default to x86 anymore, so we now need to specify its architecture.

https://github.com/dotnet/corefx/blob/master/src/Native/Windows/gen-buildsys-win.bat#L24